addons/l10n_fi/data/tax_tags.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="301-sales24" model="account.account.tag">
+            <field name="name">301-sales24</field>
+            <field name="applicability">taxes</field>
+        </record>
```

Comment:
Do these numbers 301-sales24 correspond to a number in the tax report?  I see 311, but not 301.  Or if you can link us to the VAT return you had in mind?  ( https://taxnewsreader.files.wordpress.com/2013/03/standard-vat-return-appnedix-9-finland.pdf )